core/controllers/admin_test.py

```diff
@@ -64,14 +64,10 @@
 BOTH_MODERATOR_AND_ADMIN_USERNAME = 'moderatorandadm1n'
 
 
-# TODO(#14419): Change naming style of Enum class from SCREAMING_SNAKE_CASE
-# to PascalCase and its values to UPPER_CASE. Because we want to be consistent
-# throughout the codebase according to the coding style guide.
-# https://github.com/oppia/oppia/wiki/Coding-style-guide
-class PARAM_NAMES(enum.Enum): # pylint: disable=invalid-name
+class ParamNames(enum.Enum):
     """Enum for parameter names."""
 
-    test_feature_1 = 'test_feature_1' # pylint: disable=invalid-name
+    TEST_FEATURE_1 = 'TEST_FEATURE_1'
```

Comment:
Do not change the values.

Reply:
Sir, I am getting this error if I don't change the value 
![image](https://user-images.githubusercontent.com/78922463/162727247-e3ce884d-6ee5-417c-9745-300d6b2ee0c2.png)

So, I have changed the value to resolve the error as it is taking `TEST_FEATURE_1` as the parameter but not `test_feature_1`

Reply:
You will need to use the `.value` param of the enum